### ISSUE TITLE
create_head fails for batch size 1

### ISSUE THREAD

**Issue by matthew-jurewicz** [fastai/fastai#2199]
https://github.com/fastai/fastai/blob/4052f7adb441ab8a00eaa807b444a4e583b6bcc7/fastai/vision/learner.py#L76

bn=True fails for batch size 1

**Comment by sgugger** [comment]
Yes, PyTorch Batchnorm layers require a batch size of more than one otherwise you don't have an std. Nothing we can do about in fastai.
